src/plugins/explore/public/components/query_panel/components/editor_stack/index.scss

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.editorArea {
+  /* stylelint-disable-next-line @osd/stylelint/no_restricted_properties */
+  flex: 1;
+  padding: 0;
+  background-color: $euiColorEmptyShade;
+  border-radius: 1px;
+
+  .promptEditorWrapper {
+    position: relative;
+
+    .promptEditor {
+      border: 1px solid $euiBorderColor;
+      border-radius: 1px;
+
+      .monaco-editor .view-overlays .current-line {
+        border: none;
+      }
+
+      .monaco-editor {
+        .margin {
+          background-color: $euiFormBackgroundColor;
+        }
+      }
+
+      .monaco-editor,
+      .monaco-editor-background,
+      .monaco-editor .inputarea.ime-input,
+      .monaco-editor .decorationsOverviewRuler {
+        background-color: $euiColorEmptyShade;
+      }
+
+      &.promptEditor--readonly {
+
+        .monaco-editor,
+        .react-monaco-editor-container {
+          background-color: $euiColorDarkShade !important;
+          opacity: 0.5;
+          pointer-events: none;
+        }
+
+        .monaco-editor .cursor {
+          display: none !important;
```

Comment:
Can we plz have a comment for why this `none` is needed and why `!important` was added?